src/kernel-install/kernel-install.c

```diff
@@ -81,7 +81,22 @@ static const char * const layout_table[_LAYOUT_MAX] = {
         [LAYOUT_OTHER] = "other",
 };
 
+typedef enum EntryType {
+        ENTRY_TYPE_BOTH,
+        ENTRY_TYPE_BLS1,
+        ENTRY_TYPE_BLS2,
+        _ENTRY_TYPE_MAX,
+        _ENTRY_TYPE_INVALID = -EINVAL,
+} EntryType;
+
+static const char * const entry_type_table[_ENTRY_TYPE_MAX] = {
+        [ENTRY_TYPE_BOTH] = "both",
+        [ENTRY_TYPE_BLS1] = "bls1",
+        [ENTRY_TYPE_BLS2] = "bls2",
+};
```

Comment:
hmm, thinking about this, this is quite close to BootEntryType in bootspec.h? maybe we should just reuse that, to minimize confusion? That would mean you'd do --entry-type=conf for type 1, and --entry-type=unified for type 2/ukis.

And to indicate both we should just set the entry type to _BOOT_ENTRY_TYPE_INVALID.